client/controller/index.web.js

```diff
@@ -43,7 +43,7 @@ export const ReduxWrappedLayout = ( { store, primary, secondary, tertiary } ) =>
 				nuxWelcome={ nuxWelcome }
 				translatorInvitation={ translatorInvitation }
 			/>
-			: <Layout focus={ layoutFocus } />
+			: <Layout focus={ layoutFocus } /* FIXME: Don't we need LayoutLoggedOut here? */ />
```

Comment:
I know. Kinda shame tho, since I hoped we'd be able to use a function like that for isorouting -- that would automatically either render `Layout` or `LayoutLoggedOut`.